report/src/App.jsx

```diff
@@ -13,14 +13,15 @@ import MtBlocksArea from './components/MtBlocksArea';
 /*The sample data will be replaced by the Get-MtHtmlReport when it runs the generation.*/
```

Comment:
What are your thoughts on starting with "safe" defaults of 0 for failed, passed, skipped, not run, and even total tests--just in case the actual report does not populate properly for any unexpected reasons.

Reply:
give me a few hours, need to run an errand.